uwblueprint/sistering
# 438-Send an Email along with creating the userInvite row

## Ticket link

Closes #438 



## Implementation description
* The createUserInvite resolver also sends an email once the userInvite is inserted into the database.



## Steps to test
1. Use this query:
```
mutation testCreateUserInvite($email: String!) {
   createUserInvite(email: $email, role: VOLUNTEER) {
     uuid
   }
}
```
And in your query variables, use whichever email you want (spam or smthn) and run the mutation, it should send this email:
![image](https://user-images.githubusercontent.com/71536798/179088142-8ba6c3f5-64ca-4b37-95fa-3c80986df1b4.png)

The link will take you to the page to create your account:
![image](https://user-images.githubusercontent.com/71536798/179088272-f4de6f8e-efd7-4606-8f0b-028dbafe22a4.png)

## Checklist
